### DISCUSSION TITLE
Do personal folder names get uploaded to github?

### DISCUSSION THREAD

**Discussion by oceanKingEth** [smartcontractkit/full-blockchain-solidity-course-py#31] [Q&A] [answered]
My Mac and my main folder name (in terminal) are personal. When I create a repository on github do those names end up in the path to  files or in the abi or anything?

**Comment by AlbertoSinigaglia** [comment, marked as answer]
Yes, you can easily check using
```
grep -iRl "folder_name" ./
```
and you will see infact that you will have some matches in some files (eg in the json file with the abi, the bytecode and so on)

However, in the `.gitignore` file autogenerated by brownie, you can find this  content:
```
__pycache__
.history
.hypothesis/
build/
reports/
.env
.vscode
```
and as you can see, the `build` folder is ignored, and so if you do something like `git add -A` those files won't be saved in the Github repo (and won't be tracked by git)

you should add here all the files/folders that you don't want to share, and if you are following the tutorial 100% and so you are using VSCode, you probably want to exclude `.vscode` also (that is the folder with all the preferences that you are storing for this project)

**Reply by oceanKingEth** [reply]
Awesome! Thanks for the detailed response. 

May I ask one more question? How about the blockchain? If I deploy a contract to Rinkeby, and let's say I get it verified by Etherscan, are those folder names viewable by public?

**Reply by AlbertoSinigaglia** [reply]
well, if they are reported on the code (e.g. imports), yes, otherwise no (afaik)

**Reply by PatrickAlphaC** [reply]
@AlbertoSinigaglia thanks for your answer! I made a quick edit.

**Reply by AlbertoSinigaglia** [reply]
@PatrickAlphaC  you're welcome, more than glad to help!

**Reply by PatrickAlphaC** [reply]
@oceanKingEth your whole path won't show in github though. For example, if you're working out of:
```
/home/oceanKingRealName/code/amazing_brownie_tutorial/
```
You'll only pass up `amazing_brownie_tutorial`
